pkg/render/node.go

```diff
@@ -985,6 +985,7 @@ func (c *nodeComponent) nodeDaemonset(cniCfgMap *corev1.ConfigMap) *appsv1.Daemo
 					ServiceAccountName:            CalicoNodeObjectName,
 					TerminationGracePeriodSeconds: &terminationGracePeriod,
 					HostNetwork:                   true,
+					DNSPolicy:                     corev1.DNSClusterFirstWithHostNet,
```

Comment:
Eek. This one is sorta scary... maybe we need to do it but I am pretty confident something is going to break. I just don't know what!

Reply:
Do we have another way to get around not being able to resolve the cluster service address for this?

Reply:
The only other way would be for the operator to query the Service, extract the IP, and put that in directly. Which honestly isn't terrible as a solution. 

But I'm down to try this and see what happens, at least to start...